canonical-web-and-design/snapcraft.io
# Add sticky footer

## Done
- Added a sticky footer when changes are present and adding functionality to revert changes
- Removed global footer

## QA
- Go to https://snapcraft-io-3496.demos.haus/admin/ahnuP3quahti9vis8aiw/members
- Make changes and check that a sticky footer appears at the bottom of the page
- Check that the page count is correct
- Use the filter and make changes and check that the changes persist
- In the footer click the revert button and check that all changes have been returned to their original state

## Issue
Fixes https://github.com/canonical-web-and-design/snap-squad/issues/1993
